Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsIgnoreHelpers.cs

```diff
@@ -4,7 +4,27 @@ namespace Leap.Unity.Interaction.PhysicsHands
 {
     // This script will prevent the physics hands helpers from being applied to your object.
     // This allows you to easily prevent important objects from being affected by the player.
-    // Note that this will not prevent your object from being collided with.
+
     public class PhysicsIgnoreHelpers : MonoBehaviour
-    { }
+    {
+        [Tooltip("This prevents the object from being collided with Physics Hands.")]
+        public bool DisableHandCollisions = true;
+
+        private void OnCollisionEnter(Collision collision)
+        {
+            if(!DisableHandCollisions)
+            {
+                return;
+            }
+
+            if(collision.gameObject != null && collision.gameObject.TryGetComponent<PhysicsBone>(out var temp))
+            {
+                foreach (var contact in collision.contacts)
+                {
+                    Physics.IgnoreCollision(temp.Collider, contact.thisCollider);
+                }
+            }
+        }
+
```

Comment:
Can this be changed so line 20-26 is encompassed in an if(DisableHandCllisions), rather than the double negative return we currently have?